src/Breadcrumb/Breadcrumb.js

```diff
@@ -15,25 +14,34 @@ Breadcrumb.propDescriptions = {
     children: 'List item (`BreadcrumbItem`) nodes.'
 };
 
-export const BreadcrumbItem = ({ url, link, name, className, ...props }) => {
+export const BreadcrumbItem = ({ url, link, name, className, children, ...props }) => {
+    const renderLink = () => {
+        if (!children && url) {
+            return (
+                <a className='fd-breadcrumb__link' href={url}>{name}</a>
+            );
+        } else if (children) {
+            return React.cloneElement(children, {
+                'className': 'fd-breadcrumb__link'
+            });
+        }
+    };
+
     const breadcrumbItemClasses = classnames(
         'fd-breadcrumb__item',
         className
     );
 
     return (
-        <BrowserRouter>
-            <li {...props} className={breadcrumbItemClasses}>
-                {link && <Link className='fd-breadcrumb__link' to={{ pathname: link }}>{name}</Link>}
-                {url && <a className='fd-breadcrumb__link' href={url}>{name}</a>}
-            </li>
-        </BrowserRouter>
+        <li className={breadcrumbItemClasses} {...props}>
```

Comment:
Although it technically won't matter in this case, we've been getting in the practice of spreading the props first and then include other props.

Reply:
Fixed